library/Versions.re

```diff
@@ -172,24 +176,17 @@ let endsWith = (~suffix, str) => {
 exception No_Download_For_System(System.NodeOS.t, System.NodeArch.t);
 
 let getCurrentVersion = () => {
-  switch (Fs.realpath(Directories.currentVersion)) {
-  | installationPath =>
+  switch%lwt (Fs.realpath(Directories.currentVersion)) {
+  | Missing(x) when x == Directories.currentVersion => Lwt.return_none
+  | Missing(_) => Lwt.return_some(Local.systemVersion)
+  | Exists(installationPath) =>
     let fullPath = Filename.dirname(installationPath);
-    Some(
+    Lwt.return_some(
       Local.{fullPath, name: Core.Filename.basename(fullPath), aliases: []},
```

Comment:
Still using `Core.Filename` here which is a problem on Windows

Reply:
Oh, I can take the changes you've made with @jordwalke's (and @reason-native's) `Path.re` and apply them here too 😄 But I don't want to make conflicts for #101

Reply:
Either you could just incorporate what I did in #101 and close my PR or we can chat on Discord to figure this out 😄